Extensions/TextInput/JsExtension.js

```diff
@@ -157,7 +166,7 @@ module.exports = {
 
       objectProperties
         .getOrCreate('fillColor')
-        .setValue(objectContent.fillColor || '255;255;255')
```

Comment:
It's a good idea not to change those default values unless it's actually useful to your feature. I'm not sure that's the case, or is it?